src/room_data.c

```diff
@@ -1028,7 +1028,7 @@ void reposition_all_bodies_in_room_on_subtile(struct Room *room, MapSubtlCoord s
         if (thing_is_dead_creature(thing))
         {
             ThingModel crkind = thing->model;
-            if (thing->byte_14)
+            if (corpse_laid_to_rest(thing))
```

Comment:
This `if` may be joined with `if` at 1028 (corpse_laid_to_rest is already checking for `thing_is_dead_creature`)